### PR TITLE
Import conductor configuration from k8s-datadog-agent-ops

### PR DESCRIPTION
**Pull request by chouquette**
### What does this PR do?

Import the conductor settings for agent beta builds & deployments.

### Motivation

Avoiding starting the deployment through a downstream pipeline, and better leverage conductor's features, some of which are tied to the repository initiating the run (such as build impact analysis)

### CODE REVIEW

**Comment by chouquette** [comment]
For simplicity, here's the source from the other repo: https://github.com/DataDog/k8s-datadog-agent-ops/blob/11f60d0e3df1fb2e724db3518473facf999648e0/service.datadog.yaml

File: service.datadog.yaml [hunk 1]
```diff
@@ -1,22 +1,185 @@
 ---
-schema-version: v2
+schema-version: v2.1
 dd-service: datadog-agent-nightly
+contacts:
+  - type: slack
+    contact: https://dd.slack.com/archives/C06PQ7H6PHP
```
Comment (spencergilbert) [inline service.datadog.yaml:6]: **💬 suggestion:** ‏Add the channel as a comment so we know where this is pointed

File: service.datadog.yaml [hunk 2]
```diff
@@ -1,22 +1,185 @@
...
   datadoghq.com/change-detection:
     source_patterns:
       - service.datadog.yaml
+---
+schema-version: v2.1
+dd-service: datadog-cluster-agent
+team: container-integrations
+contacts:
+  - type: slack
+    contact: https://dd.slack.com/archives/C4TQDFK1P
```
Comment (spencergilbert) [inline service.datadog.yaml:96]: Same as above

File: service.datadog.yaml [hunk 3]
```diff
@@ -1,22 +1,185 @@
...
+integrations:
+  pagerduty:
+    service-url: https://datadog.pagerduty.com/service-directory/P55U2GM
+---
+schema-version: v2.1
+dd-service: datadog-agent-cluster-worker
+team: container-integrations
+contacts:
+  - type: slack
+    contact: https://dd.slack.com/archives/C4TQDFK1P
```
Comment (spencergilbert) [inline service.datadog.yaml:144]: Same as above

**Review by spencergilbert — APPROVED** [review]

**Review by f4usto — APPROVED** [review]
/lgtm

File: service.datadog.yaml [hunk 4]
```diff
@@ -1,22 +1,188 @@
...
           ci_pipeline: "//fake_placeholder:fake_placeholder"
           branch: "main"
+          schedule: "10 3 * * SUN-THU"
+          workflows:
+            - "k8s-datadog-agent-ops/workflows/beta_builds.agents_nightly.staging-deploy"
+            - "k8s-datadog-agent-ops/workflows/beta_builds.agents_nightly.staging-validate"
+            - "k8s-datadog-agent-ops/workflows/beta_builds.agents_nightly.prod-wait-business-hours"
+            - "k8s-datadog-agent-ops/workflows/beta_builds.agents_nightly.prod-deploy"
+            - "k8s-datadog-agent-ops/workflows/beta_builds.agents_nightly.prod-validate"
+            - "k8s-datadog-agent-ops/workflows/beta_builds.agents_nightly.publish-image-confirmation"
```
Comment (KSerrania) [inline service.datadog.yaml:66]: Do we plan on moving the workflows themselves to the `datadog-agent` repository at some point?

Comment (KSerrania) [reply service.datadog.yaml:66]: Also: in the current state, does this workflow still create a `k8s-datadog-agent-ops` pipeline, or does it directly create a `datadog-agent` pipeline?

Comment (chouquette) [reply service.datadog.yaml:66]: I *think* the workflow are not meant to be moved but I'd rather get a confirmation from @sblumenthal 

This conductor target does not create a pipeline on `k8s-datadog-agent-ops`, but the conductor run on that repo still creates a downstream pipeline in `datadog-agent` and will keep doing so until we completely move conductor from their repo to here.

**Review by amenasria — APPROVED** [review]
LGTM

**Pull request merged**